### DISCUSSION TITLE
store report pages as separate html files

### DISCUSSION THREAD

**Discussion by isaacn-lab** [datapane/datapane#174] [Ideas]
Hello,

In general, I find it very convenient that the report is saved as a singe file html document.

However, sometimes the report size might become too big (in one of my use case I have a report with 60 pages, each page is app. 30 mb) and the browser cannot handle it.

My current solution is to generate a single file html document for each page using datapane and load them from auto generated index page using iframe. I couldn't find a way how to embed a static html in datapane (there are some js security measures that prevent it), so I had to write the index page myself (which works well, but doesn't look that nice).

It would be very nice if this feature (generate individual html files for each page) was available natively in datapane and could be enabled by setting corresponding flag when generating a report.

Thank you in advance!

-Nikita

**Comment by mands** [comment]
Hi @ZKpot ,

That's a good idea and shouldn't be too difficult to implement - we'll add it to the backlog. 👍

**Reply by isaacn-lab** [reply]
Great! Looking forward to using this feature, thank you!

**Reply by isaacn-lab** [reply]
Hello @mands, could you kindly advice me if there is an issue corresponding to this thread, so I can keep an eye on it?
Thank you in advance!
-Nikita

**Reply by shahinrostami** [reply]
Hi @ZKpot - this sounds like a great enhancement!

We're in the process of moving our backlog items to something that's public-facing. I'll create an issue from this discussion so we can keep an eye on it moving forward!

**Reply by isaacn-lab** [reply]
Thank you very much!

**Reply by shahinrostami** [reply]
Hey @ZKpot! Just an update to let you know we've launched the [Datapane Community forum](https://forum.datapane.com)!

There's a "Feature Requests" category there for users to request, discuss, and vote on features! I'm going to move all open issues tagged as enhancements there soon, but if you'd like to add yours instead you should get the notifications 📢

**Update:** Now available here https://forum.datapane.com/t/saving-multiple-pages-to-individual-html-files/23 👍

**Reply by isaacn-lab** [reply]
Hello, thank you for the info!
